lib/peep.rb

```diff
@@ -0,0 +1,35 @@
+
+
+
+class Peep
+  attr_reader :time, :title, :body, :username
+
+  def initialize(title, body, username)
+    @time = Time.now
+    @title = title
+    @body = body
+    @username = username
+  end
+
+  def self.get
+    @posts = DBhelper.select
+    posts_arr = []
+    @posts.each do |post|
```

Comment:
You could simplify the creation of an empty array which you modify by changing the `each` method to a `map`

`each` is a method which will perform an action on each item of the list
`map` is a method which applies a _transformation_ of each item of the list and then returns the new list.